tools/src/test/resources/1.6/valid-component-identifiers-1.6.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.6",
+  "serialNumber": "urn:uuid:3e671687-395b-41f5-a30f-a58921a69b79",
+  "version": 1,
+  "components": [
+    {
+      "type": "library",
+      "group": "com.example",
+      "name": "acme-library",
+      "version": "1.0.0",
+      "cpe": "cpe:2.3:a:example:acme-library:1.0.0:*:*:*:*:*:*:*",
+      "purl": "pkg:maven/com.example/acme-library@1.0.0",
+      "omnibor": "gitoid:blob:sha1:261eeb9e9f8b2b4b0d119366dda99c6fd7d35c64",
+      "swhid": "swh:1:cnt:94a9ed024d3859793618152ea559a168bbcbb5e2"
```

Comment:
I think we need to update docs to list the procedure involved in figuring out the swhid and omnibor id.